### PR TITLE
Find similar incidents - dateutil

### PR DESCRIPTION
**Pull request by erezh31**
fixes https://github.com/demisto/etc/issues/15575

### CODE REVIEW

File: Scripts/script-FindSimilarIncidents.yml [hunk 1]
```diff
@@ -425,3 +420,7 @@ outputs:
 scripttarget: 0
 timeout: 300ns
 runonce: false
+dockerimage: demisto/dateutil
```
Comment (anara123) [inline Scripts/script-FindSimilarIncidents.yml]: this can break backward compatibility - lets discuss this

**Comment by Itay4** [comment]
@erezh31 @anara123 whats the status here?

**Comment by erezh31** [comment]
waiting for next version because of backward compatibility, right @anara123 ?

**Comment by anara123** [comment]
@erezh31 correct

**Comment by meirwah** [comment]
@anara123 now we can merge right?

**Comment by anara123** [comment]
@erezh31 because the docker feature was merged as part of 4.5 then you should create duplicate playbook.
1. old playbook with toversion: 4.1
2. new playbook with fromversion: 4.5

**Comment by erezh31** [comment]
@anara123 why do we need one more playbook? It's exactly the same script...

**Comment by anara123** [comment]
Sorry not one more playbook, I mean one more script with fromversion 4.5

Sent from my iPhone

On 28 Mar 2019, at 13:47, erezh31 <notifications@github.com<mailto:notifications@github.com>> wrote:


@anara123<https://github.com/anara123> why do we need one more playbook? It's exactly the same script...

**Comment by erezh31** [comment]
new PR: 
https://github.com/demisto/content/pull/3312

**Pull request closed without merging**